Sources/SwifterSwift/UIKit/UIViewExtensions.swift

```diff
@@ -580,6 +627,16 @@ public extension UIView {
         return ancestorView(where: { $0 is T }) as? T
     }
 
+    /// SwifterSwift: Search constraints for this view until we find one for the given attribute.
+    ///
+    /// - Parameter attribute: the attribute to find
+    /// - Returns: matching constraint
+    func findConstraint(attribute: NSLayoutConstraint.Attribute) -> NSLayoutConstraint? {
+        allConstraints.first {
+            ($0.firstAttribute == attribute && $0.firstItem as? UIView == self) ||
```

Comment:
All constraints already make this filter on == self right? Or am I missing something?

Reply:
This is a bit subtle. While looking for .height, it's entirely possible that firstAttribute is .height but firstItem is not self (secondItem would be self in that case). That's not what we want to return.

I'll add a comment.